contracts/eth/nearbridge/contracts/NearBridge.sol

```diff
@@ -21,33 +21,35 @@ contract NearBridge is INearBridge, AdminControlled {
         uint256 stakeThreshold;
     }
 
-    // Whether the contract was initialized.
-    bool public initialized;
     uint256 public lockEthAmount;
     // lockDuration and replaceDuration shouldn't be extremely big, so adding them to an uint64 timestamp should not overflow uint256.
     uint256 public lockDuration;
     // replaceDuration is in nanoseconds, because it is a difference between NEAR timestamps.
     uint256 public replaceDuration;
     Ed25519 immutable edwards;
```

Comment:
Not sure if this the correct slot for `edwards`.

Reply:
As it neither affects the performance nor the readability, I'd rather leave it as is.